devices/ikea.js

```diff
@@ -1073,4 +1140,27 @@ module.exports = [
             }]);
         },
     },
+    {
+        zigbeeModel: ['SYMFONISK sound remote gen2'],
+        model: 'E2123',
+        vendor: 'IKEA',
+        description: 'SYMFONISK sound remote gen2',
+        fromZigbee: [ikea.fz.battery, fz.legacy.E1744_play_pause, ikea.fz.ikea_track_click, ikea.fz.ikea_volume_click,
+            ikea.fz.ikea_volume_hold, ikea.fz.ikea_dots_click_v1, ikea.fz.ikea_dots_click_v2],
+        exposes: [e.battery().withAccess(ea.STATE_GET), e.action(['toggle', 'track_previous', 'track_next', 'volume_up',
+            'volume_down', 'volume_up_hold', 'volume_down_hold', 'dots_1_initial_press', 'dots_2_initial_press',
+            'dots_1_long_press', 'dots_2_long_press', 'dots_1_short_release', 'dots_2_short_release', 'dots_1_long_release',
+            'dots_2_long_release', 'dots_1_double_press', 'dots_2_double_press'])],
+        toZigbee: [tz.battery_percentage_remaining],
+        ota: ota.zigbeeOTA,
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint1 = device.getEndpoint(1);
+            const endpoint2 = device.getEndpoint(2);
+            const endpoint3 = device.getEndpoint(3);
+            await reporting.bind(endpoint1, coordinatorEndpoint, ['genOnOff', 'genLevelCtrl', 'genPollCtrl']);
```

Comment:
The stock firmware (1.0.012) only has endpoint1 - to avoid an error with the stock firmware, `reporting.bind` for endpoints 2 and 3 should only be called if they are present, eg
```js
if (endpoint2) { 
  await reporting.bind(endpoint2, ...
}
```